docs/2.develop/relayers/relayer-gas-example.md

```diff
@@ -0,0 +1,152 @@
+---
+id: relayer-gas-example
+title: Multi-chain Relayer and Gas Station example
```

Comment:
Lets update to "Multichain" so that we are consistent everywhere

Reply:
fixed 👍